src/kt_class.h

```diff
@@ -5,20 +5,30 @@
 #include <core/reference.h>
 #include "java_instance_wrapper.h"
 #include "kt_object.h"
+#include "kt_function.h"
 
 class KtClass : public JavaInstanceWrapper {
 public:
     StringName name;
     StringName super_class;
 
     KtClass(jni::JObject p_wrapped, jni::JObject& p_class_loader);
-    ~KtClass() = default;
+    ~KtClass();
 
     KtObject* create_instance(jni::Env& env, const Variant** p_args, int p_arg_count, Object* p_owner);
+    KtFunction* get_method(const StringName& methodName);
+    KtClass* get_parent_class() const;
+    KtClass* get_native_base() const;
```

Comment:
Please drop this for now, I assume you implemented it for the methods in `KotlinScript` but those methods don't map anything in the JVM side.